src/plugins/template/backend/ops/identity.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/reference/Identity.hpp"
+
+#include "Identity_shape_inference.hpp"
+#include "evaluate_node.hpp"
+
+template <ov::element::Type_t ET>
+inline bool evaluate(const std::shared_ptr<ov::op::v15::Identity>& op,
+                     ov::TensorVector& outputs,
+                     const ov::TensorVector& inputs) {
+    using T = typename ov::element_type_traits<ET>::value_type;
+
+    const std::vector<ov::PartialShape> input_shapes{op->get_input_shape(0)};
+    const auto total_size = get_shape_size(out_shape);
+    const auto total_size_in_bytes = total_size * inputs[0].get_dtype().get_element_size();
```

Comment:
This will not work correctly for low-precision as ov element type returns 1-byte for low precision.

[Here](https://github.com/praasz/openvino/blob/cddcfe83c6f77cb48443907180dea97c22f6d625/src/core/dev_api/openvino/core/type/element_iterator.hpp#L641) is helper to calculate byte-size.

Reply:
Thank you, used the helper!